src/sentry/backup/imports.py

```diff
@@ -120,17 +120,26 @@ def _import(
             for obj in serializers.deserialize("json", src, stream=True, use_natural_keys=False):
                 o = obj.object
                 if o._meta.app_label not in EXCLUDED_APPS or o:
-                    if o.get_relocation_scope() in allowed_relocation_scopes:
+                    if o.get_possible_relocation_scopes() & allowed_relocation_scopes:
```

Comment:
why the logical `&` here?

Reply:
is `get_possible_relocation_scopes` a new function?

Reply:
It was introduced in the previous change that added `RelocationScope.Config`.

The logical and tests for set intersection. This is saying "only continue if at least one of the possible relocation scopes is allowed". If none of them are, we know that future calls to `o.get_relocation_scope()` cannot possibly return a valid scope, so we bail out early.